### ISSUE TITLE
Valet share not rewriting host

### ISSUE THREAD

**Issue by FrittenKeeZ** [laravel/valet#841]
I recently switched my setup from using Docker to Valet, and I'm having problems sharing a site.
While the tunnel does work, all links have the host `mysite.test:60` instead of the ngrok proxy.

I've tried to look for any solution, but I have no idea where to start - trying to unsecure the site and run `ngrok http mysite.test` manually doesn't work, which used to work when I was using Docker.

Anyone who can help get this resolved?
I'm using Laravel Valet 2.5.2 with ngrok 2.3.35 on macOS Catalina 10.15.1.

**Comment by drbyte** [comment]
```sh
cd ~\Sites
laravel new foobar
cd foobar
valet open
```
This shows a normal new Laravel install.

Replace the `welcome.blade.php` with just a couple links, to prove that links work:
```blade
<!DOCTYPE html>
<html>
    <body>
        <div>
            <a href="{{ url('/home') }}">Home</a>
            <a href="{{ route('test') }}">Test</a>
        </div>
    </body>
</html>
```
and add routes for both links to `web.php`:
```diff
Route::get('/', function () {
    return view('welcome');
});
+Route::get('/home', function() {
+    return 'Home Page';
+});
+Route::get('/test', function() {
+    return 'Test Page';
+})->name('test');
```

Run `valet share`

Ngrok gives a temporary URL: `http://c600469c.ngrok.io` which I CMD+clicked from terminal to open in browser.

The "Home" and "Test" links show up, and clicking them takes me to `http://c600469c.ngrok.io/home` and `http://c600469c.ngrok.io/test` respectively.
No port-60 in the URLs.

I think the clues to the solution to your situation are in whatever your app is doing differently from these basics.

**Comment by drbyte** [comment]
@FrittenKeeZ did these steps help?

**Comment by FrittenKeeZ** [comment]
Hey @drbyte,

I haven't had the time to test it yet, but will shortly - I'm suspecting it might have something to do with using `valet link` for sites instead of `valet park`, but I'll know once I've tested it.

**Comment by drbyte** [comment]
@FrittenKeeZ checking whether this still needs to be kept open. If not, please click Close.

**Comment by FrittenKeeZ** [comment]
@drbyte I've set up a notification to check it this Saturday, so hopefully I'll figure it out then.

**Comment by drbyte** [comment]
@FrittenKeeZ note that there were some updates to the Valet Share features in recent releases. While they are unlikely to have an effect on your situation, you might want to run `composer global update` and `valet install` to be sure you're using the latest.

**Comment by FrittenKeeZ** [comment]
@drbyte the issue seems to have been resolved between version `2.5.2` and `2.7.1` 👍
